build_stage1.sh

```diff
@@ -68,7 +68,15 @@ ostree remote add --repo=/srv/repo centos-atomic-host --set=gpg-verify=false htt
 
 ## compose a new tree, based on defs in centos-atomic-host.json
 
-rpm-ostree compose --repo=${OstreeRepoDir} tree ${GitDir}/centos-atomic-host.json |& tee ${BuildDir}/log.compose
+rpm-ostree compose --repo=${OstreeRepoDir} tree --touch-if-changed=${BuildDir}/changed.stamp ${GitDir}/c
+
+## stop the script unless a new commit has been made
+
+if ! test -f ${BuildDir}/changed.stamp; then
+   exit 1
```

Comment:
Minor: should probably print something here?

Reply:
👍